cristian-bars/challenge/precurse/Calculadora/calculator-pro.js

```diff
@@ -0,0 +1,79 @@
+function operaciones(opera){
+    let numberList = [];
+    let newNumber;
+    do {
+        newNumber = prompt('Enter a number or press cancel to stop');
+        if (newNumber == ""){
+            alert("Dejaste en blanco el primer numero");
+        }else if (isNaN(newNumber)){
+            alert("Entraste letras en vez de numeros");
+            newNumber = "";
+        }else if (newNumber !== null){
+            console.log(newNumber);
+            numberList.push(Number(newNumber))
+        }
+        
+        console.log(numberList);
+    } while(newNumber !== null)
+    
+    if (numberList.length == 1){
+        console.log("La raíz cuadrada del numero " + numberList[0] + " es " + Math.sqrt(numberList[0]).toFixed(3));
+    }else{
+        sumar(numberList);
+        restar(numberList);
+        multiplicar(numberList);
+        dividir(numberList);
+        
+    }
+    
+    var repeat = prompt("New numbers? y/n");
```

Comment:
Better replace 'var' with 'let'.

'var' could be deprecated in the future, also presents performance issues and invites to do bad practices.

Reply:
Done